IrcCommands/IsPlaying.py

```diff
@@ -0,0 +1,95 @@
+import re
+from Utils import config
+from Utils import pp
+from Utils import apiReq
+from Utils import roundString as roundString
+
+#Called when pm'd with
+#	/np during a map
+def run(user, msg, ircClient):
+	conf = config.Config('bot.conf')
+	irc = ircClient
```

Comment:
can just delete this line and make the function be `def run(user, msg, irc):`
(same in other `IrcCommands` files)